examples/simple/library.cc

```diff
@@ -0,0 +1,30 @@
+#include "opentelemetry/trace/provider.h"
+
+namespace trace = opentelemetry::trace;
+namespace nostd = opentelemetry::nostd;
+
+static nostd::shared_ptr<trace::Tracer> get_tracer()
```

Comment:
Optional: wrap in an anonymous namespace instead of "static"

Reply:
Done.